src/tools/knowledge_graph_tools.py

```
@@ -0,0 +1,586 @@
+"""
+Knowledge graph tools for the MCP Crawl4AI RAG server.
+
+This module contains tools for AI hallucination detection and repository analysis:
+- parse_github_repository: Parse GitHub repos into Neo4j knowledge graph
+- check_ai_script_hallucinations: Validate Python scripts against knowledge graph
+- query_knowledge_graph: Interactive Neo4j knowledge graph queries
+"""
+
+import json
+import os
+import sys
```

Ruff (0.11.9)

`sys` imported but unused

Remove unused import: `sys`

(F401)